src/Directory.Build.props

```
@@ -25,6 +25,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="PolySharp" Version="1.10.0" Condition="'$(TargetFramework)' != 'net5' AND '$(TargetFramework)' != 'net6' AND '$(TargetFramework)' != 'net7'">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Can simplify this a bit:
```suggestion
      <IncludeAssets>build; analyzers</IncludeAssets>
```

---

This is a default behavior when VS adds dev-only package. I would not change this.